src/ScaffoldInstallerPlugin.php

```diff
@@ -273,6 +273,15 @@ private function installCICommands(): void
                     }
                 }
             }
+            foreach ($this->extra['drainpipe']['tugboat'] as $tugboat) {
+                if ($tugboat === 'acquia') {
```

Comment:
What do you think about renaming `$tugboat` to `$provider`?

Reply:
Yeah you are right at the end they are providers and we can have as many other providers. Perfect makes sense thank you, I will do it.